include/boost/geometry/strategies/cartesian/distance_pythagoras.hpp

```diff
@@ -105,6 +105,24 @@ public :
                 typename calculation_type<Point1, Point2>::type
             >::apply(p1, p2);
     }
+
+    template <unsigned int Index, typename Point1, typename Point2>
+    inline typename calculation_type<Point1, Point2>::type
+    coordinate(Point1 const& p1, Point2 const& p2)
+    {
+        BOOST_CONCEPT_ASSERT( (concepts::ConstPoint<Point1>) );
+        BOOST_CONCEPT_ASSERT( (concepts::ConstPoint<Point2>) );
+
+        assert_dimension_equal<Point1, Point2>();
+
+        return geometry::get<Index>(p1) - geometry::get<Index>(p2);
+    }
+
+    template <typename CT>
+    inline CT vertical_or_meridian(CT const& lat1, CT const& lat2) const
```

Comment:
Similar story here. There are no meridians in cartesian. It looks like you wanted to move responsibility for some calculations from Seg-Box or Pt-Seg strategy. Are `vertical_or_meridian()` or `coordinate()` needed in distance Pt/Pt algorithm?